.env.sample

```diff
@@ -7,3 +7,4 @@ DB_PASSWORD=
 DB_HOST=
 
 JWT_SECRET_KEY= secret_key
+SENDGRID_API_KEY=
```

Comment:
Don't forget to check the configuration to automatically add an empty line at the end of every file.
[https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)